bin/fin

```diff
@@ -4732,7 +4732,7 @@ _exec ()
 	local SHELL_NONINTERACTIVE="sh -lc"
 
 	# For cli use bash
```

Comment:
Add a comment around why bash has to be used for `db`

Reply:
done